### PR TITLE
Updated syntax

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
Updated the syntax of templates and queries to better reflect stream worker's capabilities.

### CODE REVIEW

File: docs/cep/store-rdbms-plugin.md [hunk 1]
```diff
@@ -42,15 +41,29 @@ The following examples assume you have an input stream called `TriggerStream` an
...
 
 ```sql
-select numRecords from TriggerStream#rdbms:cud("SAMPLE_DB", "UPDATE Customers_Table SET customerName='abc' where customerName='xyz'") 
-insert into  RecordStream;
+INSERT INTO RecordStream
+SELECT numRecords 
+FROM TriggerStream#rdbms:cud("jdbc:mysql://hostname:3306/MySQLDB?useSSL=false", "username", "password", "com.mysql.jdbc.Driver", "UPDATE Customers_Table SET customerName='abc' where customerName='xyz'");
+```
+
+Note: Make sure the stream is previously defined with:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Make sure the stream is defined with:
```

File: docs/cep/store-rdbms-plugin.md [hunk 2]
```diff
@@ -42,15 +41,29 @@ The following examples assume you have an input stream called `TriggerStream` an
...
 
 ```sql
-select numRecords from TriggerStream#rdbms:cud("SAMPLE_DB", "UPDATE Customers_Table SET customerName=? where customerName=?", changedName, previousName) 
-insert into  RecordStream;
+INSERT INTO RecordStream
+SELECT numRecords 
+FROM TriggerStream#rdbms:cud("jdbc:mysql://hostname:3306/MySQLDB?useSSL=false", "username", "password", "com.mysql.jdbc.Driver",  "UPDATE Customers_Table SET customerName=? where customerName=?", changedName, previousName);
+```
+
+Note: Make sure the stream is previously defined with:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Make sure the stream is defined with:
```

File: docs/cep/store-rdbms-plugin.md [hunk 3]
```diff
@@ -76,10 +89,9 @@ Additionally, the `attributeName` attribute (any type) returns the attributes li
 Use the following template to create a Procedure function:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Use the following template to create a procedure function:
```

File: docs/cep/store-rdbms-plugin.md [hunk 4]
```diff
@@ -89,17 +101,31 @@ In these examples, the values in the parentheses for `RETURNCON()` are the input
...
 
 ```sql
-select Name, Age, Date_Time from IntrimStream#rdbms:procedure('ORACLE_DB', 'Name String, Age int,Date_Time String', 'begin RETURNCON(?,?); end;','cursor', NoOfYears)
-insert into tempStream1;
+INSERT INTO tempStream1
+SELECT Name, Age, Date_Time
+FROM IntrimStream--#rdbms:procedure('ORACLE_DB', 'Name String, Age int,Date_Time String', 'begin RETURNCON(?,?); end;','cursor', NoOfYears);
+```
+
+Note: Make sure the stream is previously defined with:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Make sure the stream is defined with:
```

File: docs/cep/store-rdbms-plugin.md [hunk 5]
```diff
@@ -89,17 +101,31 @@ In these examples, the values in the parentheses for `RETURNCON()` are the input
...
 
 ```sql
-select Name, Age, Date_Time from IntrimStream#rdbms:procedure('ORACLE_DB', 'Name String, Age int,Date_Time String', 'begin RETURNCON(9,?); end;','cursor')
-insert into tempStream1;
+INSERT INTO tempStream1
+SELECT Name, Age, Date_Time
+FROM IntrimStream--#rdbms:procedure('ORACLE_DB', 'Name String, Age int,Date_Time String', 'begin RETURNCON(9,?); end;','cursor');
+```
+
+Note: Make sure the stream is previously defined with:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Make sure the stream is defined with:
```

File: docs/cep/store-rdbms-plugin.md [hunk 6]
```diff
@@ -127,36 +153,51 @@ Additionally, `attributeName` (any type) returns the attributes listed in the pa
...
 
 ```sql
-select creditcardno, country, transaction, amount from TriggerStream#rdbms:query('SAMPLE_DB', 'creditcardno string, country string,transaction string, amount int', 'select * from where country=?', countrySearchWord)  
-insert into recordStream;
+INSERT INTO recordStream
+SELECT creditcardno, country, transaction, amount
+FROM TriggerStream#rdbms:query("jdbc:mysql://hostname:3306/MySQLDB?useSSL=false", "username", "password", "com.mysql.jdbc.Driver", 'creditcardno string, country string,transaction string, amount int', 'select * from where country=?', countrySearchWord);
+```
+
+Note: Make sure the stream is previously defined with:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Make sure the stream is defined with:
```

File: docs/cep/store-rdbms-plugin.md [hunk 7]
```diff
@@ -127,36 +153,51 @@ Additionally, `attributeName` (any type) returns the attributes listed in the pa
...
 ```sql
-select creditcardno, country, transaction, amount from TriggerStream#rdbms:query('SAMPLE_DB', 'creditcardno string, country string,transaction string, amount int', 'select * from where country=?', countrySearchWord, true)  
-insert into recordStream;
+INSERT INTO recordStream 
+SELECT creditcardno, country, transaction, amount
+FROM TriggerStream#rdbms:query("jdbc:mysql://hostname:3306/MySQLDB?useSSL=false", "username", "password", "com.mysql.jdbc.Driver", 
+    'creditcardno string, country string,transaction string, amount int', 'select * from where country=?', countrySearchWord, true);
+```
+
+Note: Make sure the stream is previously defined with:
```
Comment (diana-macrometa) [inline docs/cep/store-rdbms-plugin.md]:
```suggestion
Make sure the stream is defined with:
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]

**Review by jesusgonzalez12 — COMMENTED** [review]
Thanks for the recommendations @diana-macrometa

**Review by diana-macrometa — APPROVED** [review]

**Pull request merged**